Bio/Phylo/PhyloXMLIO.py

```diff
@@ -488,6 +495,7 @@ def _parse_taxonomy(self, parent):
         return taxonomy
 
     def other(self, elem, namespace, localtag):
+        """Set non-phyloXML element."""
```

Comment:
Is "set" the right word here? In the context of Python properties that verb is used to assign a new value.

(Same comment applies to the following methods too)